js/iteration-task.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+function iterativeOddSumTo(number){
+    let oddSum = 0;
+    for(let i = 0; i <= number; i++){
+        (i % 2 === 1 && Math.sign(i) === 1) ? oddSum += i : null;
```

Comment:
Тут правий операнд тернарного оператора не потрібен, null тільки для того щоб синтаксис був вірний. Можна зробити простіше:
```suggestion
        (i % 2 === 1 && Math.sign(i) === 1) && (oddSum += i);
```

Reply:
І потім, якщо нам потрібно все що поверне оператор %, крім 0, то можемо ще спростити запис:
```suggestion
        (i % 2 && Math.sign(i) === 1) && (oddSum += i);
```

Reply:
І потім, для якого випадку нам потрібно перевіряти що `i` від'ємне число, якщо в тебе заданий старт зі значення `0` і далі ти тільки інкрементиш це значення?

Тому ми можемо піти ще далі, і спростити рішення до:
```suggestion
        i % 2 && (oddSum += i);
```